src/fusedav.h

```diff
@@ -20,3 +20,7 @@
 ***/
 
 #endif
+
+// Part of https://getpantheon.atlassian.net/browse/FS-1165
+#define TOPPLING_VALUE 100000
+#define BLOCKING_THRESHOLD TOPPLING_VALUE-1000
```

Comment:
Could we replace these with a single macro?  There is no set/fixed hard limit; the actual number will vary depending on specifics of each volume.  We don't gain anything by using an obscure number (such as 99,000 dirents).  I expect a hard limit of 100k to be enough to prevent most if not all problems, and it matches our already-published documentation.

I'd like the name to be very clear/obvious - maybe something like:
```suggestion
#define MAX_DIRENTS_IN_DIR 100000
```

We can debate the usefulness of having separate warning and blocking thresholds elsewhere/later; I'm going to argue that would not be useful because 1) nobody is regularly looking at fusedav logs, 2) this problem isn't big/common enough to justify developing a mechanism to proactively/automatically do that, and 3) this situation is generally caused by automated resizing, where such an early warning wouldn't make any meaningful difference to allow user response anyway (i.e., the hard cutoff would likely be reached a few seconds after the warning triggered).

Reply:
https://github.com/pantheon-systems/fusedav/pull/518

Reply:
Changed.